clouddrift/utility.py

```diff
@@ -0,0 +1,211 @@
+"""
+This module contains various utility functions.
```

Comment:
I get the long-term vision but while there's only one function let's be specific and precise.

```suggestion
This module provides a function to easily and efficiently plot trajectories stored in a ragged array.
```